examples/02-plot/image_depth.py

```diff
@@ -0,0 +1,41 @@
+"""
+Render a depth image
+~~~~~~~~~~~~~~~~~
+
+Plot a depth image as viewed from a camera overlooking the "hills" example mesh.
+"""
+import numpy as np
+import pyvista as pv
+import matplotlib.pyplot as plt
+from pyvista import examples
+
+# Load an interesting example of geometry
+mesh = examples.load_random_hills()
+
+# Establish geometry within a pv.Plotter()
+pv.close_all()
+p = pv.Plotter()
+p.add_mesh(mesh, color=True)
+p.show(auto_close=False)
```

Comment:
We need to be sure to close the plotter (`p.close()`) after we are done utilizing it

Reply:
Done.  See 19c2b594767cc8cab3ab495822111b90b94473ab